ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -192,6 +205,22 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
     );
   }
 
+  const hasWalletEip155 = requestedCaipChainIds.includes('wallet:eip155');
+  const hasOtherEip155Scopes = requestedCaipChainIds.some(
+    (caipChainId) =>
+      caipChainId.startsWith('eip155:') && caipChainId !== 'wallet:eip155',
+  );
+
+  let supportedRequestedCaipChainIds = _supportedRequestedCaipChainIds;
+  if (hasWalletEip155 && !hasOtherEip155Scopes) {
```

Comment:
are we using `wallet:eip155` by itself as a way to determine if this request came from the eip-1193 provider? if so we should use the metadata field `isEip1193Request` instead